dev-ruby/nokogiri/nokogiri-1.11.1-r1.ebuild

```diff
@@ -25,10 +25,12 @@ IUSE=""
 RDEPEND="${RDEPEND}
 	>=dev-libs/libxml2-2.9.10:=
 	>=dev-libs/libxslt-1.1.34
+	sys-libs/zlib
 	virtual/libiconv"
 DEPEND="${DEPEND}
 	>=dev-libs/libxml2-2.9.10
 	>=dev-libs/libxslt-1.1.34
+	sys-libs/zlib
```

Comment:
remove and fix the RDEPEND include

Reply:
I don't want to mess with the (strange) patterns like this which are in a lot of Ruby ebuilds